src/ui/public/directives/__tests__/input_number.js

```diff
@@ -0,0 +1,49 @@
+import angular from 'angular';
+import expect from 'expect.js';
+import ngMock from 'ng_mock';
+import 'ui/directives/input_whole_number';
```

Comment:
import input_number and not input_whole_number?  Interesting, I wonder how this is working.